libcloudforensics/gcp.py

```diff
@@ -337,14 +463,10 @@ def CreateDiskFromSnapshot(self,
     return GoogleComputeDisk(
         project=self, zone=self.default_zone, name=disk_name)
 
-  def GetOrCreateAnalysisVm(self,
-                            vm_name,
-                            boot_disk_size,
-                            disk_type='pd-standard',
-                            cpu_cores=4,
-                            image_project='ubuntu-os-cloud',
-                            image_family='ubuntu-1804-lts',
-                            packages=None):
+  def GetOrCreateAnalysisVm(
+      self, vm_name, boot_disk_size, disk_type='pd-standard', cpu_cores=4,
+      image_project='ubuntu-os-cloud', image_family='ubuntu-1804-lts',
+      packages=None):
```

Comment:
With this many params, I'd rather have them one per line. (same for the other ones)

Reply:
agree, but our YAPF config automagically 'improved' this. Reverted back to one per line.